### PR TITLE
#3 Implement basic obstecale avoidence.

### LINKED ISSUE #3: Drive forward
### **Description**

As a driver, I want the car to be able to move forward, so that I can move,

### **Acceptance criteria**

Upon request, the car moves forward.

### PR DESCRIPTION
**Pull request by majawennroth**
This commit makes the car drive forward and stop 15 cm befor hitting an obstecale. solves: #8

### CODE REVIEW

File: car/Obstical_Avoidence/Obstical_Avoidence.ino [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  Serial.begin(9600);
+  
+}
+
+void loop() {
+  
+  int distance = front.getDistance();
+
+  if (distance <= 15){ //stop when distance is less than 15 cm.
```
Comment (platisd) [inline car/Obstical_Avoidence/Obstical_Avoidence.ino]: If there is no obstacle, then distance will be `0`. What should your car do then?

File: car/Blink/Blink.ino [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  pinMode(LED_BUILTIN, OUTPUT);
+}
+
+// the loop function runs over and over again forever
+void loop() {
+  digitalWrite(LED_BUILTIN, HIGH);   // turn the LED on (HIGH is the voltage level)
+  delay(1000);                       // wait for a second
+  digitalWrite(LED_BUILTIN, LOW);    // turn the LED off by making the voltage LOW
+  delay(1000);                       // wait for a second
+}
```
Comment (platisd) [inline car/Blink/Blink.ino]: Remove this file

**Review by platisd — CHANGES_REQUESTED** [review]

**Comment by majawennroth** [comment]
@platisd the changes you requested is now in the latest commit.

**Comment by platisd** [comment]
> @platisd the changes you requested is now in the latest commit.

There's a little "refresh" icon next to my username. If you press on it a review will be re-requested. :+1:

**Review by platisd — APPROVED** [review]

**Review by platisd — APPROVED** [review]

**Pull request merged**